docs/api/qiskit/0.19/qiskit.validation.BaseSchema.mdx

```diff
@@ -26,7 +26,7 @@ python_api_name: qiskit.validation.BaseSchema
     type
   </Attribute>
 
-  ###
+  ### constructor is passed all named parameters from deserialization.
 
   <Attribute name="constructor is passed all named parameters from deserialization." />
 
```

Comment:
This section also feels like an error. I'm not too worried about it as it's an obscure class from an ancient version of Qiskit, but if we can work out why it happened then it'll make the script more robust

Reply:
Agreed! I noticed it too, but that's the same heading we had before changing to MDX components as you can see [here](https://github.com/Qiskit/documentation/blob/0406030c7e13a828bc5bd164b9ccb7aac29f4b06/docs/api/qiskit/0.19/qiskit.validation.BaseSchema.mdx?plain=1#L32). My plan was to investigate it in a follow-up, I think it could be similar to [#1120](https://github.com/Qiskit/documentation/issues/1120) with the `model\_cls` attributes just above of it (it seems like it's missing description matching the heading)

Reply:
Ok I see, that's fine then; thanks

Reply:
Might be helpful to open an issue with the problem so we don't forget it